idx/memory/memory.go

```diff
@@ -158,64 +175,79 @@ func (t *TagIndex) delTagId(name, value string, id schema.MKey) {
 	}
 }
 
-// org id -> nameWithTags -> Set of references to schema.MetricDefinition
-// nameWithTags is the name plus all tags in the <name>;<tag>=<value>... format.
-type defByTagSet map[uint32]map[string]map[*schema.MetricDefinition]struct{}
+// defByTagSet stores pointers to idx.MetricDefinitions
+//
+// org id -> NameWithTagsHash() -> Set of references to idx.MetricDefinition.
+// NameWithTagsHash() returns an Md5 sum stored in a struct of two uint64s. The hash is derived from
```

Comment:
it's not up to the docs of defByTagSet to document how idx.Md5Hash works internally.

Reply:
this is a way in which docs become redundant and out of date.